compiler/rustc_interface/src/passes.rs

```diff
@@ -336,6 +337,20 @@ fn early_lint_checks(tcx: TyCtxt<'_>, (): ()) {
     )
 }
 
+fn env_var(tcx: TyCtxt<'_>, key: Symbol) -> Option<Symbol> {
```

Comment:
I was unsure where to put the query definition, so I just put it in the same file as some other general queries.